### ISSUE TITLE
only decoding ?

### ISSUE THREAD

**Issue by tst2005** [exosite/lua-yaml#6]
Hello,

I'm searching a lua implementation for `yamldata = yaml.encode(luadata)` & `luadata = yaml.decode(yamldata)`.

Is you current lua-yaml implementation only support decoding (with `require"yaml".eval(yamldata)`) ?

Best Regards,

**Comment by dominicletz** [comment]
Yep, sorry it does only do decoding. Encoding though should be much easier to implement compared...
